provision/virtualbox.sh

```diff
@@ -3,6 +3,4 @@
 set -e
 set -x
 
-sudo pacman -S --noconfirm linux-headers
-sudo pacman -S --noconfirm virtualbox-guest-utils-nox
-sudo systemctl enable vboxservice
```

Comment:
According to https://wiki.archlinux.org/index.php/VirtualBox#Load_the_VirtualBox_kernel_modules_2
>Since version 5.0.16, virtualbox-guest-modules-arch and virtualbox-guest-dkms use systemd-modules-load service to load their modules at boot time.